libraries/Microsoft.Bot.Builder/Adapters/TestAdapter.cs

```diff
@@ -20,9 +20,11 @@ public class TestAdapter : BotAdapter
         private object _activeQueueLock = new object();
 
         private int _nextId = 0;
+        private readonly bool sendTraceActivity;
 
-        public TestAdapter(ConversationReference conversation = null)
+        public TestAdapter(ConversationReference conversation = null, bool sendTraceActivity = false)
```

Comment:
Don't have argument names the same as member variable names. In this case, change the member variable to have a leading "_" to follow our convention.